doc/wave_driver.html

```diff
@@ -165,10 +165,13 @@
 </blockquote>
 <p dir="ltr">-t [--traceto] path</p>
 <blockquote> 
-  <p dir="ltr">Enable the tracing facility build into the <tt>Wave</tt> library. 
+  <p dir="ltr">Enable the tracing facility built into the <tt>Wave</tt> library.
     The path specifies the filename to use for the output of the generated trace 
     log. If the filename given equals to <tt>'-'</tt> (without the quotes), the 
     trace log is put into the standard error stream (stderr).</p>
+  <p dir="ltr">Tracing is only performed for portions of the input where it has been
+    explicitly enabled via the <tt>#pragma wave trace(enable)</tt> directive.
```

Comment:
Perhaps we should explicitly mention the inline `_Pragma` version as it's more useful for this kind of thing:
```suggestion
    explicitly enabled via the <tt>#pragma wave trace(enable)</tt> or <tt>_Pragma("wave trace(enable)")</tt> directive.
```